doc/api.md

```diff
@@ -174,7 +174,7 @@ Create a new [Proposal](#proposal). The root key is the client model slug, e.g.
 ```bash
 % curl -i -X POST -H 'Content-Type: application/json' \
   -H 'Authorization: Bearer your-access-token' \
-  --data @proposal.json \
+  --data proposal.json \
```

Comment:
yes this is very possible! In the end I was unable to actually test this `curl` because of my problems with the auth token authorization. So was a guess.